Sources/Socket.swift

```diff
@@ -2769,7 +2769,7 @@ public class Socket: SocketReader, SocketWriter {
 	}
 	
 	// MARK: --- UDP
-	
```

Comment:
Looks like my editor trimmed some whitespace... Apologies for the superfluous edits.